src/message.rs

```diff
@@ -638,6 +638,20 @@ impl Message {
         None
     }
 
+    pub async fn get_videochat_url(&self) -> Option<String> {
+        if self.viewtype == Viewtype::VideochatInvitation {
+            self.param.get(Param::VideochatUrl).map(|s| s.to_string())
+        } else {
+            None
+        }
+    }
+
+    pub fn is_basic_videochat(&self) -> bool {
+        // currently, all videochat-urls are of type basic-webrtc
```

Comment:
so this function is not future proof?

Reply:
the function is future-proof - once, we decide to support jitsi or so, we can adapt the implementation so it will return false when jitsi and not basic-webrtc is used.

Reply:
the function is gone, there is now the clearer get_videochat_type()